src/main/java/org/radarcns/security/filter/AuthenticationFilter.java

```diff
@@ -69,7 +85,7 @@ private static synchronized TokenValidator getValidator() {
             if (mpUrlString != null) {
                 try {
                     YamlServerConfig cfg = new YamlServerConfig();
-                    cfg.setResourceName("res_RestApi");
+                    cfg.setResourceName("res_restApi");
```

Comment:
if this is configurable in csv, maybe better to read it from a config too.

Reply:
having it fixed for this particular client seems fine though.